### DISCUSSION TITLE
Allow Measurable Habits to be Skipped

### DISCUSSION THREAD

**Discussion by Kackerlacka** [iSoron/uhabits#1335] [Feature Requests] [answered]
I use skips on my "Yes or No" habits a lot, but sometimes I want to skip a Measurable habit, and it doesn't currently seem possible. For example, I fast 16 hrs during the week but not weekends. I'd like to have a Measurable habit to input the hours each day, but also be able to skip during the weekends without having to enter 0 and affect my score for something that's not part of my goal.

**Comment by iSoron** [comment, marked as answer]
Thanks for the suggestion, @Kackerlacka. This has been implemented by @kalina559 in #1319 and should become available in Loop 2.1.

**Reply by pseudacris** [reply]
When is 2.1 being released?
